nannou/Cargo.toml

```diff
@@ -15,8 +15,8 @@ default = ["notosans"]
 
 [dependencies]
 cgmath = { version = "0.17", features = ["serde"] }
-conrod_core = "0.72"
-conrod_wgpu = "0.72"
+conrod_core = "0.73"
```

Comment:
I think we need to update this in `nannou_timeline` too

Reply:
Oh yes, there's that timeline example that is not building :(

Reply:
Done